src/protocolsupport/protocol/packet/middleimpl/clientbound/play/v_pe/StopSound.java

```diff
@@ -0,0 +1,31 @@
+package protocolsupport.protocol.packet.middleimpl.clientbound.play.v_pe;
+
+import protocolsupport.api.ProtocolVersion;
+import protocolsupport.protocol.ConnectionImpl;
+import protocolsupport.protocol.packet.ClientBoundPacket;
+import protocolsupport.protocol.packet.middle.serverbound.play.MiddleStopSound;
```

Comment:
hmm weird, wonder why this middle packet is under serverbound

Reply:
Yeah, I have no idea, I think I did that because PC's was in the server one, but it seems like it doesn't make a difference so I changed it to clientbound.